### ISSUE TITLE
FormBlocListener onLoading never called

### ISSUE THREAD

**Issue by aaassseee** [GiancarloCode/form_bloc#178]
By following, the tutorial on [https://giancarlocode.github.io/form_bloc/#/loading_and_initializing_example](https://giancarlocode.github.io/form_bloc/#/loading_and_initializing_example). With switching from BlocBuilder to FormBlocListener, onLoading function within FormBlocListener will never call.

**Comment by LaysDragon** [comment]
yeah,I get the same problem, its seems like because the first state would never trigger the bloc listiner ,so the onLoading would never working here,it bring me some problem to handle the init trigger problem for loading logic, now I have to switch other way to implemented this
